aws-py-oidc-provider-pulumi-cloud/README.md

```diff
@@ -0,0 +1,93 @@
+# Provisioning an OIDC Provider in AWS for Pulumi Cloud
+
+This example will create OIDC configuration between Pulumi Cloud and AWS, specifically demonstrating connectivity with [Pulumi ESC](https://www.pulumi.com/docs/pulumi-cloud/esc/). The program automates the process detailed in the AWS documentation for the following activities:
+
+- [Obtaining the thumbprint for an OpenID Connect Identity Provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html)
+- [Creating an OpenID Connect Identity Provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html)
+
+## Prerequisites
+
+* [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+* [Configure Pulumi to Use AWS](https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/) (if your AWS CLI is configured, no further changes are required)
+
+## Running the Example
+
+Clone [the examples repo](https://github.com/pulumi/examples/tree/master/aws-py-oidc-provider) and navigate to the folder for this example.
+
+```bash
+git clone https://github.com/pulumi/examples.git
+cd examples/aws-py-oidc-provider-pulumi-cloud
+```
+
+Next, to deploy the application and its infrastructure, follow these steps:
+
+1. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init dev
+    ```
+
+1. Set your desired AWS region:
+
+    ```bash
+    pulumi config set aws:region us-east-1 # any valid AWS region will work
+    ```
+
+1. Install requirements.
+
+    ```bash
+    python -m venv venv
+    source venv/bin/activate
+    pip3 install -r requirements.txt
+    deactivate
```

Comment:
Just double checking, do we want to deactivate the virtual env? (I'm not a python expert)

Reply:
Yes it doesn't need to be active for the program to run as Pulumi behind the scenes magic knows to check in the venv folder. I actually can simplify this script a bit more though.

Reply:
Gotcha!!!